docs/articles/documentation/test-api/selecting-page-elements/framework-specific-selectors.md

```diff
@@ -86,6 +64,27 @@ await t.expect(listAngular.testProp).eql(1);
 
 To learn more, refer to the [plugin repository](https://github.com/DevExpress/testcafe-angular-selectors/blob/master/angular-selector.md).
 
+## AngularJS
+
+`AngularJSSelector` contains a set of static methods to search for an HTML element by the specified binding (`byModel`, `byBinding`, etc.).
+
+```js
+import { AngularJSSelector } from 'testcafe-angular-selectors';
+import { Selector } from 'testcafe';
+
+fixture `TestFixture`
+    .page('http://todomvc.com/examples/angularjs/');
+
+test('add new item', async t => {
+    await t
+        .typeText(AngularJSSelector.byModel('newTodo'), 'new item')
+        .pressKey('enter')
+        .expect(Selector('#todo-list').visible).ok();
+});
+```
+
+To learn more, see the [angularJS-selector.md](https://github.com/DevExpress/testcafe-angular-selectors/blob/master/angularJS-selector.md) file in the plugin repository.
```

Comment:
Let's do the same `To learn more` phrase for AngularJS and Angular sections.

AngularJS - To learn more, see the angularJS-selector.md file in the plugin repository.
Angular - To learn more, refer to the plugin repository.

Reply:
I don't think it matters actually, but if you like...